trino/dbapi.py

```diff
@@ -517,6 +531,27 @@ def fetchmany(self, size=None) -> List[List[Any]]:
 
         return result
 
+    def describe(self, sql: str) -> List[DescribeOutput]:
+        """
+        Returns the schema of a given SQL statement
```

Comment:
It returns not only schema, please rephrase it.

Reply:
Done